.github/workflows/release.yml

```diff
@@ -64,7 +64,7 @@ jobs:
 
   build:
     name: Build Release Binaries
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```

Comment:
Some discussion related to this, because a large number of users will still be on older versions of Ubuntu, and IIRC, compiling on newer versions will break packages for those; https://github.com/containerd/containerd/pull/10405#issuecomment-2670901505

Reply:
This change should be fine though right as we are continuing to build within a ubuntu-20.04 container or is somehow libseccomp dependency coming from the host?

Reply:
This shouldn't change the output and we will test the configuration

Reply:
> This change should be fine though right as we are continuing to build within a ubuntu-20.04 container

Is this something that we should be doing. ubuntu-20.04 will go EOL in April 2025.  Similar reason why we upgraded from ubuntu 18.04 to ubuntu 20.04 in https://github.com/containerd/containerd/issues/7961

Reply:
Good point, does anyone know if Canonical's extended security support extends to their container images? I assume so. If which case, containerd 1.6 will go EOL in July and containerd 1.7 next March.